.github/workflows/build-latest.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       protocol_version_default: 21
       xdr_ref: v21.1.0
       core_ref: v21.3.1
-      horizon_ref: horizon-v2.32.0
+      horizon_ref: horizon-v22.0.0-rc1
```

Comment:
@leighmcculloch , can the `latest` variant refer to rc's or should it wait for 'ga' release, i.e. `horizon-v22.0.0` ?

Reply:
Latest should wait for stable releases.